rofi_rbw/rofi_rbw.py

```diff
@@ -49,17 +49,20 @@ def main() -> None:
 
 
 def get_data(name: str, folder: str) -> Data:
+    command = ['rbw', 'get', '--full', '--folder', folder, name]
```

Comment:
I would suggest to first define the minimal command, and in the `if` block add the two folder parameters to the list. This way, there is less duplication.